packages/core/src/http/requestBuilder.ts

```diff
@@ -205,13 +206,15 @@ export class DefaultRequestBuilder<BaseUrlParamType, AuthParams>
   protected _authParams?: AuthParams;
   protected _retryOption: RequestRetryOption;
   protected _apiErrorFactory: ApiErrorFactory;
+  protected _apiLogger: ApiLogger;
   public prepareArgs: typeof prepareArgs;
 
   constructor(
     protected _httpClient: HttpClientInterface,
     protected _baseUrlProvider: (arg?: BaseUrlParamType) => string,
     protected _apiErrorCtr: ApiErrorConstructor,
     protected _authenticationProvider: AuthenticatorInterface<AuthParams>,
+    protected _loggerOptions: LoggingOptions,
```

Comment:
Instead of taking the LoggingOptions as a parameter to create the ApiLogger instance, inject the ApiLogger instance directly. The SDK client initialization code will then be responsible for creating the ApiLogger instance and passing it to the request builder.